TTHAnalysis/python/plotter/ttH-multilepton/mca-includes/mca-2lss-CP-splitdecays.txt

```diff
@@ -1,20 +1,20 @@
-ttH_cosa_1p00_q_1p00_hww+ : TTH_ctcvcp  : xsec*LHEReweightingWeight[11]  : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge && (GenHiggsDecayMode==24); FillColor=ROOT.kOrange+10, genSumWeightName="LHEReweightingSumw_[11]/LHEScaleSumw_[4]*genEventSumw_"
```

Comment:
@ataliercio I argue that in order to be consistent the -CP txt files should be left as they are. The changes should be better applied to new -diff txt files :)